hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -326,13 +329,17 @@ public void setCurrentCompactionLog(long latestSequenceNum) {
     appendToCurrentCompactionLog("");
   }
 
+
   @Override
-  public void close() {
-    synchronized (this) {
-      if (!closed) {
-        closed = true;
-        if (executor != null) {
-          executor.shutdown();
+  public void close() throws Exception {
```

Comment:
throws Exception is not required I guess here.

Reply:
It is not required. It was added because I used IDE to add this method and `AutoCloseable#close()` throws an exception. I leave it as it is unless there is a major concern about it. Also we are using `IOUtils.closeQuietly()` so exception will be logged and swallowed.